packages/elvis-illustrations/package.json

```diff
@@ -5,6 +5,7 @@
   "license": "MIT",
   "type": "module",
   "exports": {
+    "./CHANGELOG.json": "./CHANGELOG.json",
```

Comment:
Is this something we need to expose through our NPM package? 🙂 I don't find it very common to export changelogs in NPM packages.

Reply:
It needs to be exposed explicitly here because we use the `exports` field to define the entry points of the package, and we need to use the changelog in the documentation page 😄 Without specifying this, it is not possible to access.